Data_Engineer_KoNoHa.ipynb

```diff
@@ -0,0 +1,2775 @@
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "Location             0\n",
+              "Price              248\n",
+              "Rooms             1706\n",
+              "Bathrooms         2013\n",
+              "Car Parks        17567\n",
+              "Property Type       25\n",
+              "Size              1063\n",
+              "Furnishing        6930\n",
+              "dtype: int64"
+            ],
+            "text/html": [
+              "<div>\n",
+              "<style scoped>\n",
+              "    .dataframe tbody tr th:only-of-type {\n",
+              "        vertical-align: middle;\n",
+              "    }\n",
+              "\n",
+              "    .dataframe tbody tr th {\n",
+              "        vertical-align: top;\n",
+              "    }\n",
+              "\n",
+              "    .dataframe thead th {\n",
+              "        text-align: right;\n",
+              "    }\n",
+              "</style>\n",
+              "<table border=\"1\" class=\"dataframe\">\n",
+              "  <thead>\n",
+              "    <tr style=\"text-align: right;\">\n",
+              "      <th></th>\n",
+              "      <th>0</th>\n",
+              "    </tr>\n",
+              "  </thead>\n",
+              "  <tbody>\n",
+              "    <tr>\n",
+              "      <th>Location</th>\n",
+              "      <td>0</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>Price</th>\n",
+              "      <td>248</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>Rooms</th>\n",
+              "      <td>1706</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>Bathrooms</th>\n",
+              "      <td>2013</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>Car Parks</th>\n",
+              "      <td>17567</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>Property Type</th>\n",
+              "      <td>25</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>Size</th>\n",
+              "      <td>1063</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>Furnishing</th>\n",
+              "      <td>6930</td>\n",
+              "    </tr>\n",
+              "  </tbody>\n",
+              "</table>\n",
+              "</div><br><label><b>dtype:</b> int64</label>"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 132
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Data Cleaning"
+      ],
+      "metadata": {
+        "id": "2y0lTqTXPqLa"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Drop missing value\n",
+        "df = df.dropna()"
```

Comment:
Carpark null value alone comprises of 31% percent of overall data. Do you think we should drop the car park column? Kalau tak drop nak buat cemana?